x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/export/get_export_rule_action_connectors.ts

```diff
@@ -42,11 +43,30 @@ const mapExportedActionConnectorsDetailsToDefault = (
     excluded_action_connections: exportDetails.excludedObjects,
   };
 };
+const filterOutPredefinedActionConnectorsIds = async (
+  actionsClient: ActionsClient,
+  actionsIdsToExport: string[]
+): Promise<string[]> => {
+  const allActions = await actionsClient.getAll();
+  const predefinedActionsIds = allActions
+    .filter(({ isPreconfigured }) => isPreconfigured)
+    .map(({ id }) => id);
+  if (predefinedActionsIds.length)
+    return actionsIdsToExport.filter((id) => !predefinedActionsIds.includes(id));
+  return actionsIdsToExport;
```

Comment:
nit: the check is unnecessary, you could just:
```
return actionsIdsToExport.filter((id) => !predefinedActionsIds.includes(id));
```
since if `predefinedActionsIds` is empty you'll just get `actionsIdsToExport`

Reply:
Thanks, Juan!!

> `since if predefinedActionsIds is empty you'll just get actionsIdsToExport`

It will create a new reference with the same elements "a shallow copy", as it is going to iterate over each element and check the condition which wasn't needed as it was initially falsy.